protected/humhub/modules/space/models/Space.php

```diff
@@ -87,6 +92,10 @@ class Space extends ContentContainerActiveRecord implements Searchable
     // Model Scenarios
     const SCENARIO_CREATE = 'create';
     const SCENARIO_EDIT = 'edit';
+    // Ethereum status
+    const ETHEREUM_STATUS_DISABLED = 0;
+    const ETHEREUM_STATUS_IN_PROGRESS = 1;
+    const ETHEREUM_STATUS_ENABLED = 2;
```

Comment:
where are these used?

Reply:
I'm using this constants in `xcoin module` & `ethereum module` to check which `eth_status` has a space at a given moment : 
- ETHEREUM_STATUS_DISABLED : Ethererum is diasbled 
- ETHEREUM_STATUS_IN_PROGRESS : only POST /wallet & POST /dao are executed 
- ETHEREUM_STATUS_ENABLED : Ethereum is fully enabled